examples/auth0/lib/auth0.js

```diff
@@ -2,7 +2,7 @@ import { initAuth0 } from '@auth0/nextjs-auth0'
 
 export default initAuth0({
   secret: process.env.SESSION_COOKIE_SECRET,
-  issuerBaseURL: process.env.NEXT_PUBLIC_AUTH0_DOMAIN,
+  issuerBaseURL: `https://${process.env.NEXT_PUBLIC_AUTH0_DOMAIN}`,
```

Comment:
Is there a reason for adding this here instead of to the environment variable directly?